### PR TITLE
Ft guest errors fix

### PR DESCRIPTION
**Pull request by offiongbassey**
All guest errors fixed.

### CODE REVIEW

File: .github/workflows/whatsapp-push-notify-action.yml
Comment (eleezs) [inline .github/workflows/whatsapp-push-notify-action.yml:1]: Why is this here?

File: package.json
Comment (eleezs) [inline package.json:1]: why this?

File: src/controllers/auth.js
Comment (eleezs) [inline src/controllers/auth.js:1]: Is this part of the fix?

File: src/helpers/validation.js [hunk 1]
```diff
@@ -3,15 +3,15 @@ import { validationResult } from "express-validator";
 import { responseHandler } from "./responseHandler";
 
 
-export const existingEmail = async (email, { req }) => {
+export const existingEmail = async (email) => {
```
Comment (eleezs) [inline src/helpers/validation.js:6]: These 2 function do same this, is it possible for you to combine them? Or abstract it

existingEmail and  verifyExistingEmail

File: src/helpers/validation.js [hunk 2]
```diff
@@ -109,3 +109,37 @@ export const validationHandler = (values = []) => {
     responseHandler(res, 422, false, { errors: errors.array() });
   };
 };
+
+
+export const existingUsername = async (username) => {
```
Comment (eleezs) [inline src/helpers/validation.js:114]: Why is this here?

File: src/middlewares/validator.js [hunk 3]
```diff
@@ -150,3 +150,100 @@ export const search_guests_validator = [
         .custom(query => checkAllowedFields(query, ['gender', 'name', 'email', 'phone', 'status']))
 
 ]
+
+export const multiple_guests_validator = [
```
Comment (eleezs) [inline src/middlewares/validator.js:154]: Again what is this doing here?

**Review by eleezs — CHANGES_REQUESTED** [review]
Kindly go through this PR, and make the required changes. I need fixes to what is on dev not new features in this PR

File: src/middlewares/validator.js (hunk 3 above)
Comment (offiongbassey) [reply src/middlewares/validator.js:154]: is from the multiple guest PR that I raised. The PR hasn't been merged yet
